### PR TITLE
[CU-1td6awj] [CU-1rpc2zt] refactoring pallets assets governance 

### PR DESCRIPTION
**Pull request by dzmitry-lahoda**
took this
[CU-1rpc2zt] pallet-assets #259
merged into main
made some invalid states unrepresentative (cleanuped some probably overenginering)
all things are as same
need this for xcmp
not yet in runtime

### CODE REVIEW

**Comment by itsbobbyzzz** [comment]
Task linked: [CU-1rpc2zt Pallet-Assets](https://app.clickup.com/t/1rpc2zt)

**Comment by itsbobbyzzz** [comment]
Task linked: [CU-1td6awj XCMP:  assets and assets-registry pallets in same branch](https://app.clickup.com/t/1td6awj)

**Review by Br1ght0ne — APPROVED** [review]
Thanks @dzmitry-lahoda

**Comment by dzmitry-lahoda** [comment]
@Br1ght0ne for fmt, please run `make style` in root. for benchamrks - never did benchmarks. seems adding benchmarks requires adding these into some list in runtime.

**Comment by hussein-aitlahcen** [comment]
@Br1ght0ne any chance we have this branch passing the CI?

**Review by hussein-aitlahcen — APPROVED** [review]

File: frame/assets/src/tests.rs [hunk 1]
```diff
@@ -0,0 +1,209 @@
+use crate::*;
+//use composable_traits::currency::MultiCurrency;
```
Comment (andor0) [inline frame/assets/src/tests.rs:2]: did you forget to remove it?

**Review by andor0 — APPROVED** [review]

File: frame/assets/src/tests.rs (hunk 1 above)
Comment (dzmitry-lahoda) [reply frame/assets/src/tests.rs:2]: will remove in next pr.

**Pull request merged**